pkg/util/utils.go

```diff
@@ -291,3 +293,31 @@ func ContainsString(slice []string, s string) bool {
 	}
 	return false
 }
+
+// ValidateUser validates minikube is run by the recommended user (privileged or regular)
+func ValidateUser(driverName string) {
+	u, err := user.Current()
+	// Check if minikube needs to run with sudo or not.
+	if err == nil {
+		if driverName == constants.DriverNone && u.Name != "root" {
+			exit.Usage("Please run with sudo. The vm-driver %q requires sudo.", constants.DriverNone)
+
+		} else if u.Name == "root" && !(driverName == constants.DriverHyperv || driverName == constants.DriverNone) {
```

Comment:
The username of "root" is meaningless in Windows (as is uid 0), so just simplify this check to not compare against HyperV.